app/controllers/users/registrations_controller.rb

```diff
@@ -4,12 +4,28 @@ module Users
   # This controller exists so that we can block user registration if local user
   # login is disabled.
   class RegistrationsController < Devise::RegistrationsController
+    before_action :configure_permitted_parameters
```

Comment:
Generally it is preferred to put a blank line between the `before_actions` and then method definitions.